youtube_dl/downloader/external.py

```diff
@@ -46,6 +52,10 @@ def exe(self):
         return self.params.get('external_downloader')
 
     @classmethod
+    def available(cls):
+        return check_executable(cls.get_basename(), cls.available_opt)
```

Comment:
I suppose it's better to wrap into list here and store bare option string in `available_opt`. This is also more idiomatic to type it in uppercase.